packages/cloud-cognitive/src/components/AboutModal/AboutModal.test.js

```diff
@@ -136,32 +111,16 @@ describe(componentName, () => {
     expect(link.href).toEqual(linkHref);
   });
 
-  it('renders legal text', () => {
-    renderComponent({ legalText, open: true });
-    screen.getByText(legalText);
+  it('renders general text', () => {
+    renderComponent({ generalText, open: true });
+    screen.getByText(generalText);
   });
 
   it('renders copyright text', () => {
     renderComponent({ copyrightText, open: true });
     screen.getByText(copyrightText);
   });
 
-  it('renders a clickable carbon tab for additional info', () => {
-    renderComponent({ additionalInfo, open: true });
-    const tabToSelect = screen.getByRole('tab', { name: tabLabel2 });
-    const tabSelected = `${carbon.prefix}--tabs__nav-item--selected`;
-    expect(tabToSelect).not.toHaveClass(tabSelected);
-    userEvent.click(tabToSelect);
-    expect(tabToSelect).toHaveClass(tabSelected);
-  });
-
-  it('renders a version number', () => {
-    renderComponent({
-      additionalInfo: [{ label: tabLabel1, content: versionNumber }],
-    });
-    screen.getByText(versionNumber);
-  });
-
```

Comment:
We should keep version number and update this test since version number is a specific element of the AboutModals.